test/api/users.coffee

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+User = require("../../website/src/models/user").model
+app = require("../../website/src/server")
+
+describe "Users", ->
+
+  describe "Deleting Users", ->
+
+    before (done) ->
+      registerNewUser ->
+        done()
+      , true
+
+    it 'Should delete a user', (done) ->
+      async.waterfall [
+        (cb) ->
+          registerManyUsers 1, cb
```

Comment:
If you pass false as the second argument to register new user, than the register new user function returns the user it creates. [See here](https://github.com/HabitRPG/habitrpg/blob/develop/test/api/api-helper.coffee#L55).